src/components/Explore/TokenDetails/TokenDetail.tsx

```diff
@@ -176,6 +177,17 @@ const TruncatedAddress = styled.span`
     display: flex;
   }
 `
+const NetworkBadge = styled.div<{ networkColor?: string; backgroundColor?: string }>`
+  display: flex;
+  border-radius: 5px;
+  padding: 4px 6px;
+  align-items: center;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 12px;
```

Comment:
isn't line height the same as font-size by default

Reply:
oh ya whoops lol